.github/workflows/test.yml

```
@@ -12,6 +12,7 @@ jobs:
         node:
           - 12
           - 14
+          - lts/*
```

Not sure if we want to point to `lts/*` instead of `v16`.
When v18 becomes LTS, does this mean we will be running tests for `12`, `14` and `18`?

---

Yes I think we should just add `16`.

We probably could write an @octoherd script to bump the node matrix when a new version becomes available :D 

```suggestion
          - 16
```

---

Agreed, let's go with 16 instead of the `lts/*`. Looks like there is some discussion going on to have additional version strings to help with this https://github.com/actions/setup-node/issues/26